src/common.c

```diff
@@ -1189,6 +1192,8 @@ ssize_t ofi_bsock_send(struct ofi_bsock *bsock, const void *buf, size_t *len)
 	if (*len > bsock->zerocopy_size) {
 		ret = bsock->sockapi->send(bsock->sockapi, bsock->sock, buf, *len,
 					   MSG_NOSIGNAL | OFI_ZEROCOPY, bsock);
+		if (ret == -FI_EINPROGRESS_URING)
+			return ret;
```

Comment:
Thanks. I will disable zerocopy with uring in the next PR.

Reply:
FYI, zerocopy send feature for io_uring was introduced in Linux version 6.

Reply:
Thanks - we'll need to add some check for this.  This isn't critical, since the user can toggle these options independently and both are off by default.